### DISCUSSION TITLE
LDAP issue for Dependency Track

### DISCUSSION THREAD

**Discussion by rinkucool007** [DependencyTrack/dependency-track#1005] [General]
Hi,
My Dependency Track site is running in docker container. I am trying setup LDAP in Dependency Track webapp. Below is the application.properties

alpine.ldap.enabled=true
alpine.ldap.server.url=ldap://ldapserver.example.com:3268
alpine.ldap.domain=example.com
alpine.ldap.basedn=dc=example,dc=com
alpine.ldap.security.auth=simple
alpine.ldap.bind.username=cn=admin,ou=XX,ou=Users,dc=exampleu,dc=example,dc=com
alpine.ldap.bind.password=secret
alpine.ldap.auth.username.format=%s
alpine.ldap.attribute.name=userPrincipalName
alpine.ldap.attribute.mail=

After restarting the container when I was trying to add LDAP user, I am getting “Syncing” and not able to add user in it. 
Did anyone faces similar issue and fixed it. Appreciate your help.

 
![image](https://user-images.githubusercontent.com/25628468/113796538-6dfbdb00-971d-11eb-9dd9-e42abf641652.png)

**Comment by stevespringett** [comment]
Related and possible duplicate of #983

**Comment by stevespringett** [comment]
That looks like a partial/incomplete configuration. 

See https://docs.dependencytrack.org/getting-started/ldap-configuration/ for examples of tested configurations against supported LDAP providers.

**Reply by rinkucool007** [reply]
Hi Steve, I have checked and used the same configuration but still it showed syncing. Is there any way to troubleshoot the issue from the log.

**Reply by stevespringett** [reply]
Enable debug logging. https://docs.dependencytrack.org/getting-started/configuration/

**Reply by rinkucool007** [reply]
In the link it was set for war file. How to setup log levels in a running container?

**Reply by stevespringett** [reply]
Per the documentation:
> For Docker deployments, simply set the LOGGING_LEVEL environment variable to one of INFO, WARN, ERROR, DEBUG, or TRACE.
